Training/1.0 Introduction.md

```diff
@@ -12,37 +12,61 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+String - 'Hello World!'
+Int - 5
+Float - 1.1
+Bool - True or False
+List - [1,2,3,4,5]
+Tuple - (1,2,3,4,5)
+Dictionary - {'apple':20, 'pear':10}
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+list
+lis = [1,2,3,4]
+arrays
+ar = np.arrays([[1,2],[3,4]])
+tuples
+tup = (1,2,3,4)
 ```
 
 ### Task 3: 
 
 We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
 ```
-
+if x>2:
+x = x+1
+else print('x is less than 2')
 ```
 
 Now incorporate a for loop to calculate the value of n(x+1) for n repetitions where n=3. Store each value of the for loop in an array named y.  
 
 ```
-
+y = []
+n = 3
+for i inrange 3:
+   x = random.randit(1,10)
+   y.append(n*(x+1))
+print(y)
 ```
 
 ### Task 4: 
 
 Finally we will have a look at creating functions. Functions allow us to compute processes much faster without having to repeat lines of code. Using your code from Task 3, create a function called 'my_cool_function', which takes in the value x and will compute the n(x+1) if x is greater than 2 and for n repetitions where values will be stored in an array named y. Your function will return y. 
 
 ```
-
+def my_cool_function(x):
+y = []
+n = 3
+for i in range(n)
+  if x>2:
+    y.append(n*(x+1))
+  return(y)
```

Comment:
Correct use of function but incorrect indentation syntax. Also see previous comments regarding numpy arrays.